src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,25 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList">
+    {
+      comments.length > 0
+        ? (
+          comments.map((comment) => (
+            <CommentInfo comment={comment} key={comment.id} />
+          ))
+        )
+        : (
+          <b data-cy="NoCommentsMessage">
+            No comments yet
+          </b>
+        )
```

Comment:
You can make it shortest
```suggestion
      comments.length
        ? comments.map((comment) => (
            <CommentInfo comment={comment} key={comment.id} />
          ))
        : (
          <b data-cy="NoCommentsMessage">
            No comments yet
          </b>
        )
```